install/helm/agones/values.yaml

```diff
@@ -126,6 +126,7 @@ agones:
       serviceType: LoadBalancer
       annotations: {}
     generateTLS: true
+    generateClientTLS: false
```

Comment:
`true` by default?
Also, test it by getting a dependency to the client secret in the e2e: https://github.com/googleforgames/agones/blob/master/test/e2e/allocator_test.go#L61-L62

Reply:
Done!

Reply:
Set it to false from below.

Reply:
Can you explain more? I am not sure what you mean.

Reply:
Changed the default value back to true and set the value to false in the Makefile.